### PR TITLE
[WIP] add DJL Android ONNXRuntime support

### PR DESCRIPTION
**Pull request by wxm2018**
## Description ##

Brief description of what this PR is about
- Add DJL Android support for the ONNXRuntime engine to enable it to use onnx format models.- 
- But now it is [WIP].

### CODE REVIEW

File: android/core/build.gradle [hunk 1]
```diff
@@ -97,11 +97,19 @@ afterEvaluate {
     }
 }
 
+repositories {
```
Comment (lanking520) [inline android/core/build.gradle:100]: This can be removed

File: android/onnxruntime-native/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+
+# ONNXRuntime Android Build Procedure
```
Comment (lanking520) [inline android/onnxruntime-native/README.md:2]: I think the installation instruction is generally applicable to all versions. You can migrate the build instruction to Android folder and refer a link from README

File: android/onnxruntime-native/build.gradle [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+
+
+
+
+
+
+
+
+//apply plugin: 'java-library'
```
Comment (lanking520) [inline android/onnxruntime-native/build.gradle:141]: This is the setup for transient onnxruntime, @frankfliu what do you think?

**Review by lanking520 — CHANGES_REQUESTED** [review]

File: android/onnxruntime-native/README.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+This procedure is meant only for developers who want to build PyTorch native binaries from source, this is not intended for regular users.
```
Comment (frankfliu) [inline android/onnxruntime-native/README.md:4]: This section is copied from PyTorch module, we need write different one for ONNX

File: android/build.gradle [hunk 5]
```diff
@@ -7,6 +7,7 @@ buildscript {
 
     dependencies {
         classpath 'com.android.tools.build:gradle:4.2.2'
+        classpath 'com.github.kezong:fat-aar:1.3.8'
```
Comment (frankfliu) [inline android/build.gradle:10]: fat-aar may not work as you expected.
engine.jar bundle in aar won't work, it need to be dexed first.

I think we should just publish a pom only package as normal java project.

File: android/core/build.gradle (hunk 1 above)
Comment (frankfliu) [inline android/core/build.gradle:100]: We should not need this.

We can use mavenLocal():
1. cd onnxruntime-native
2. gradle pTML

**Comment by lanking520** [comment]
As discussed offline. Let's just rollback to the previous trasient dependency version

File: android/onnxruntime-native/build.gradle (hunk 3 above)
Comment (frankfliu) [reply android/onnxruntime-native/build.gradle:141]: Please try my PR: https://github.com/deepjavalibrary/djl/pull/1866

Your android project should be able to simply include:
```
implementation "ai.djl.android:onnxruntime:0.19.0-SNAPSHOT"
```

**Review by frankfliu — APPROVED** [review]

**Review by lanking520 — APPROVED** [review]

**Pull request merged**